docs/python/examples/stock-tracker.md

```diff
@@ -49,7 +51,8 @@ def fetch_stock_prices_workflow(scheduled_time: datetime, actual_time: datetime)
 
 ## Fetching stock prices
 
-We'll use the `yfinance` library to fetch stock prices and write a DBOS [step](../tutorials/step-tutorial) to make it durable.
+Let's write a function that fetches stock prices using the `yfinance` library.
+We annotate it with [`DBOS.step`](../tutorials/step-tutorial.md) to call it from the durable workflow above.
```

Comment:
One doesn't need to annotate a function to make it callable by a workflow. (That's unfortunately one way to read this sentence.)